### DISCUSSION TITLE
帮我看下为什么nas下的服务访问自身另一个服务会走clash核心？如何绕过呢？

### DISCUSSION THREAD

**Discussion by livebelive** [vernesong/OpenClash#3644] [Q&A]
![image](https://github.com/vernesong/OpenClash/assets/59411705/df556fc5-d7cb-47ff-9867-6a15f081ac04)
感谢回答！我用的是fake-ip tun混合模式
图上192.168.2.5这个IP并不是openwrt本身的，我的op是装在另一台主机上的，IP是192.168.2.253，并且这个192.168.2.5的群晖网关并没有指向openwrt，而是在docker里面开的HTTP_PROXY
还有一个奇怪的现象，图上的日志中，有时候目标地址会变成我的dns...但我找遍了我用的几个服务，也没找到哪里设置了访问域名，局域网内的服务互访都是用的局域网本地IP

**Comment by ghost** [comment]
插件设置-流量控制-本机代理

**Reply by livebelive** [reply]
感谢回答！我用的是fake-ip tun混合模式
图上192.168.2.5这个IP并不是openwrt本身的，我的op是装在另一台主机上的，IP是192.168.2.253，并且这个192.168.2.5的群晖网关并没有指向openwrt，而是在docker里面开的HTTP_PROXY

**Reply by livebelive** [reply]
还有一个奇怪的现象，有时候目标地址会变成我的dns...

**Comment by livebelive** [comment]
再更新下情况，我的群晖nas 192.168.2.5  现在网关和DNS都没有经过op，但是核心的日志 还会有这个记录，而且非常多，一会日志就满了，为什么会经过核心呢？
